static/app/views/profiling/profileSummary/index.tsx

```diff
@@ -686,9 +695,27 @@ function ProfileDigest() {
     sort: {key: 'last_seen()', order: 'desc'},
     referrer: 'api.profiling.profile-summary-table',
   });
-
   const data = profiles.data?.data?.[0];
 
+  const latestProfile = useProfileEvents<ProfilingFieldType>({
+    cursor: profilesCursor,
+    fields: ['profile.id', 'last_seen()'],
```

Comment:
You should `timestamp` instead of `last_seen()` here because it will create an expensive group by

Reply:
Ahh, forgot we had that! Thank you!